src/package.h

```diff
@@ -11,10 +11,17 @@ namespace cyclus {
 
 /// Packager is a class that packages materials into discrete items in ways that mimic realistic nuclear material handling. Packages will eventually be a required parameter of resources.
 class Package {
-
+  friend class Context;
   public:
     typedef boost::shared_ptr<Package> Ptr;
 
+    // create a new package type. Should be called by the context only
+    // (see Context::AddPackage), unless you want an untracked package
+    //  type (which you probably don't)
+    static Ptr Create(std::string name, double fill_min = 0,
+                      double fill_max = std::numeric_limits<double>::max(),
+                      std::string strategy);
```

Comment:
All arguments with defaults have to come after those without.  I suggest a default for `strategy = "first"`.

Reply:
oops missed that one i guess